src/Neo.CLI/config.fs.mainnet.json

```diff
@@ -40,7 +40,8 @@
       "HF_Aspidochelone": 3000000,
       "HF_Basilisk": 4500000,
       "HF_Cockatrice": 5800000,
-      "HF_Domovoi": 5800000
+      "HF_Domovoi": 5800000,
+      "HF_Echidna": 0
```

Comment:
It's hard to plan when we're going to have the next release. May be we can postpone configuration changes until some real proximity to the release and in this PR add only `HF_Echidna` enum?

Reply:
> It's hard to plan when we're going to have the next release. May be we can postpone configuration changes until some real proximity to the release and in this PR add only `HF_Echidna` enum?

Its placeholder, can be updated before merge to the master.

Reply:
This will make configs stored in the `master` branch unusable for real synchronization for some time, nothing good about that. Better omit it, declare `HF_Echidna` (we need it and will use it), but update configs later.

Reply:
> This will make configs stored in the `master` branch unusable for real synchronization for some time, nothing good about that. Better omit it, declare `HF_Echidna` (we need it and will use it), but update configs later.

this will not be merged to the master branch until we decides to release new version.